src/transformStateWithClones.js

```diff
@@ -7,7 +7,36 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let lastStateVersion = state;
```

Comment:
use one variable for a new state
const newState = Object.assign({}, state);

Reply:
Agree